src/operand/reference.ts

```diff
@@ -86,11 +87,27 @@ function contextValueLookup(ctx: Context, key: string): Result {
   return undefined
 }
 
+/**
+ * Converts a value to a specified data type, returns original value if not parseable.
+ * @param value value to parse as data type
+ */
+function toDataType(value: Result, dataType: string | undefined): Result {
+  switch (dataType) {
+    case 'Number':
+      return toNumber(value)
+    case 'String':
+      return toString(value)
+    default:
+      return value
```

Comment:
Should we consider throwing an error here?

Reply:
Maybe not throwing but console.warn would be nice! To give us the chance to fix these

Reply:
b8f7162